data/tools/abaplint.yml

```diff
@@ -0,0 +1,10 @@
+name: abaplint
+category:
+  - linter
+tags:
+  - abap
+license: MIT License
+type: []
```

Comment:
Will we have to fill out the type for each tool before we merge? If so, I'd suggest to pre-fill it with "cli" (which should be the most common type, I guess) to make the manual editing quicker.